docs/content/Dual Analyzer.fsx

```diff
@@ -115,6 +115,48 @@ let editorAnalyzer (ctx: EditorContext) : Async<Message list> =
 Both analyzers will follow the same code path: the console application will always have the required data, while the editor needs to be more careful.  
 ⚠️ Please do not be tempted by calling `.Value` on the `EditorContext` 😉.
 
+To enable a wide range of analyzers, both context types give access to very detailed information about the source code.  
+Among this information is the full untyped abstract syntax tree (AST) and the typed abstract syntax tree (TAST).
+As you can deduce from the example above, processing these trees is a very common task in an analyzer. But writing your own tree traversal code can be daunting and can also get quite repetitive over many analyzers.  
+That's why the SDK offers the `ASTCollecting` and `TASTCollecting` modules. In there, you'll find facility types and functions to make your analyzers author life easier.
+For both trees, a type is defined, `SyntaxCollectorBase` and `TypedTreeCollectorBase` respectively, with members you can override to have easy access to the tree elements you want to process.  
```

Comment:
Add links to API docs: http://localhost:8901/reference/fsharp-analyzers-sdk-astcollecting-syntaxcollectorbase.html